Docker-compose/docker-compose.yml

```diff
@@ -61,6 +61,8 @@ services:
     container_name: satosa-saml2spid
     depends_on:
       - satosa-mongo
+    stdin_open: true   # enables PDB when attach to the compose
```

Comment:
you could add this configuration among the environments:
```
stdin_open: ${SATOSA_DEBUG:-false}
tty: ${SATOSA_DEBUG:-false}
```
with this option we can add the `honour-stdin` option in satosa start if `$SATOSA_DEBUG` if true.